api/src/application/application.service.ts

```diff
@@ -125,6 +125,11 @@ export class ApplicationService {
     application.assignedTo = user;
     application.lastUpdatedBy = loggedInUser;
     application.lastUpdatedByUserGuid = loggedInUser.userGuid;
+    if (application.status?.name === ApplicationStatus.RECEIVED) {
```

Comment:
I see that status changes but it doesn't refresh after assigning evaluator. Whereas when they click on "Proceed to the next step" it redirects them back to applications table.